app/graphql/query_root.rb

```diff
@@ -0,0 +1,15 @@
+QueryRoot = GraphQL::ObjectType.define do
+  name "QueryRoot"
+
+  field :me, ApiUser::Type do
+    resolve ->(obj, args, ctx) { ctx[:api_user] if ctx[:api_user].logged_in? }
+  end
+
+  field :organization do
+    type OrganizationType
+    argument :id, !types.ID, "Filter by organization ID"
+    resolve ->(obj, args, ctx) {
+      Organization.find(args[:id])
```

Comment:
Shoud this check the org is public until we figure out authorization scopes?
`Organziation.where(id: args[:id], listed: true)`